Library/test-helpers/lib.sh

```diff
@@ -1166,32 +1178,44 @@ limeWaitForAgentStatus() {
     [ -n "$3" ] && TIMEOUT=$3
 
     for I in `seq $TIMEOUT`; do
-        keylime_tenant -c status -u $UUID &> $OUTPUT
-	if grep -E -q "\"operational_state\": \"$STATUS\"" $OUTPUT; then
-            cat $OUTPUT
-	    rm $OUTPUT
-	    return 0
-	fi
-        sleep 1
+	    if [ -n "$NAME" ]; then
```

Comment:
Since you are not declaring `NAME` as `local` always, it could be non-empty if such a variable is defined by a user. Better always declare `local` variable (preferably with different name, more on that above).